packages/soci-snapshotter/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "soci-snapshotter"
+version = "0.1.0"
+publish = false
+build = "../build.rs"
+edition = "2021"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+releases-url = "https://github.com/awslabs/soci-snapshotter/releases"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/awslabs/soci-snapshotter/archive/refs/tags/v0.5.0.tar.gz"
```

I prefer using URLs of this form so that the downloaded file has the expected name, rather than just `v0.5.0.tar.gz`:
```suggestion
url = "https://github.com/awslabs/soci-snapshotter/archive/v0.5.0/soci-snapshotter-0.5.0.tar.gz"
```